.codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+
+  status:
+    project: yes
+    patch: yes
+    changes: no
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: false  # if true: only post the comment if coverage changes
```

Comment:
I like that this will only post a comment [once](https://docs.codecov.io/docs/pull-request-comments) and then update it. I don't think coveralls did that. Still, I would kind of prefer no comment at all. It shows up in githubs CI listing anyway, what value does the additional comment add?

Reply:
I think it could be useful to see at a glance how the coverage for a pull request changed. Sure it’s available with two clicks in the CI listing, but the information is more scattered on the codecov website.
Additionally, I think it is easy to ignore in all other cases, where it is not needed.

Reply:
I don't think the usefulness justifies the verbosity. Its only really relevant when the coverage meaningfully decreases, which hopefully will be the minority of the time (and then the big red x should be obvious enough).

I made the comment at `require_changes` because I think `require_changes = true` would be a nice compromise if you prefer to keep the comment. I don't have a very strong preference either way :)

Reply:
I can see the concern, which is why I would propose monitoring the usefulness over time. I will merge it as is for now.